pkg/artifactory/resource/security/resource_artifactory_scoped_token.go

```diff
@@ -207,7 +207,7 @@ func (r *ScopedTokenResource) Schema(ctx context.Context, req resource.SchemaReq
 				MarkdownDescription: "Also create a reference token which can be used like an API key. Default is `false`.",
 				Optional:            true,
 				Computed:            true,
-				Default:             booldefault.StaticBool(false),
+				//Default:             booldefault.StaticBool(false),
```

Comment:
Let's remove rather than comment out this line

Reply:
You are too fast :) it's already removed.